<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="diffusion_planner_param_path" default="$(find-pkg-share autoware_diffusion_planner)/config/diffusion_planner.param.yaml"/>

  <node pkg="autoware_diffusion_planner" exec="autoware_diffusion_planner_node" name="diffusion_planner_node" output="screen">
    <param from="$(var diffusion_planner_param_path)" allow_substs="true"/>
    <remap from="~/output/trajectory" to="/planning/scenario_planning/lane_driving/trajectory"/>
    <remap from="~/input/odometry" to="/localization/kinematic_state"/>
    <remap from="~/input/acceleration" to="/localization/acceleration"/>
    <remap from="~/input/route" to="/planning/mission_planning/route"/>
    <remap from="~/input/traffic_signals" to="/perception/traffic_light_recognition/traffic_signals"/>
    <remap from="~/input/tracked_objects" to="/perception/object_recognition/tracking/objects"/>
  </node>
</launch>
